ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/clientlibs/site/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[core.forms.components.numberinput.v1.runtime]"
+    dependencies="[core.forms.components.runtime]"/>
```

Comment:
deps needs to be correct.  core.forms.components.runtime.base, container

Reply:
made the suggested change